docker-compose.yaml

```diff
@@ -47,6 +47,7 @@ services:
       - DB_USERNAME=postgres
       - DB_PASSWORD=postgres
       - SELF_ADDRESS=5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty
+      - AUTH_TYPE=${IDENTITY_AUTH_TYPE:-NONE}
```

Comment:
just occured to me we might want only a single `DEPENDENCY_AUTH_TYPE` env since it's unlikely we'll ever have deps running different auth types to one another?

Reply:
combined, assumed deps always run same `AUTH_TYPE`